src/main/java/io/confluent/connect/jdbc/source/DataConverter.java

```diff
@@ -265,13 +330,22 @@ private static void addFieldSchema(ResultSetMetaData metadata, int col,
         } else {
           builder.field(fieldName, Schema.BYTES_SCHEMA);
         }
+//        if (hasDefault) {
```

Comment:
delete these?

Reply:
should I be handling this case?

Reply:
If they have a default byte value set, then yes I would handle it. Is there a complexity I am missing here?

Reply:
Not extra complexity, I was thrown off because a blob field in mysql cannot have a default value. Looks like the binary ones can so I'll add that.